docs/phdata.rst

```diff
@@ -397,6 +407,60 @@ The following fields are also arrays:
   This field is not relevant when no polarization- and spectral-insensitive
   splitting is performed.
 
+The following fields can be used to indicate the meaning of space-time markers.
+
+- **num_space_time_markers**: (integer) *New in version 0.6.* number of
```

Comment:
We should specify either this field is not included for non-space-time measurements, or that if there are no space-time markers, this field must be 0 if specified.

Reply:
Doesn't the following text already accomplish this? (The value must match the number of markers, namely zero, if there are no markers; and the field is not required in that case.)